docs/reference/changelog-r2021.md

```diff
@@ -20,6 +20,7 @@ Released on June, Xth, 2021.
     - Add an example that shows an integration of OpenAI Gym with Webots ([#2711](https://github.com/cyberbotics/webots/pull/2711)).
     - Added a simple room with a Nao robot ([#2701](https://github.com/cyberbotics/webots/pull/2701)).
   - New Features:
+    - Display magnitude of velocity/rotational speed vectors, to help monitoring robot displacement ([#3209](https://github.com/cyberbotics/webots/pull/3209)).
```

Comment:
This is an enhancement and not a new major feature. Could you please move it in the `Enhancements` section?
Moreover we should use the correct terminology as displayed in the UI:
```suggestion
    - Display magnitude of linear and rotational velocity vectors in the [Field Editor](../guide/the-scene-tree.md#field-editor), to help monitoring robot displacement ([#3209](https://github.com/cyberbotics/webots/pull/3209)).
```